packages/react/src/themes/teams/components/Text/textVariables.ts

```diff
@@ -8,6 +19,7 @@ export interface TeamsTextVariables extends TextVariables {
 
 export default (siteVariables): Partial<TeamsTextVariables> => {
   return {
+    // TODO: how can we specify these typings here, when this is already defined and used in the base theme!?
```

Comment:
@kuzhelov this is the problem I was talking about (having different typings for the color schemes in base vs teams theme. How can we address this?

Reply:
We may use generic for the color scheme typing, not sure how will it go with the overriding, as we are using this in the styles...

Reply:
discussed with @mnajdova and agreed to introduce generic type for variables in the base theme, with default type value of `ColorSchemeMapping`:

```js
// base theme
export interface TextVariables<TColorSchemeMapping = ColorSchemeMapping> { ... }
```

This will allow client to restrict/modify declarations of schema types in those cases where it is necessary, and `ColorSchemeMapping` is a good type to capture general cases.